src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Services/AssessorDashboardOrchestrator.cs

```diff
@@ -31,20 +31,32 @@ public async Task AssignApplicationToAssessor(Guid applicationId, int assessorNu
             await _apiClient.AssignAssessor(applicationId, new AssignAssessorApplicationRequest(assessorNumber, assessorUserId, assessorName));
         }
 
-        private void AddApplicationsToViewModel(NewApplicationsViewModel viewModel, List<RoatpAssessorApplicationSummary> applications)
+        public async Task<InProgressApplicationsViewModel> GetInProgressApplicationsViewModel(string userId)
+        {
+            var applicationSummary = await _apiClient.GetAssessorSummary(userId);
+            var applications = await _apiClient.GetInProgressApplications(userId);
+
+            var viewModel = new InProgressApplicationsViewModel(userId, applicationSummary.NewApplications, applicationSummary.InProgressApplications, applicationSummary.ModerationApplications, applicationSummary.ClarificationApplications);
+            AddApplicationsToViewModel(viewModel, applications);
+            return viewModel;
+        }
+
+        private void AddApplicationsToViewModel(DashboardViewModel viewModel, List<RoatpAssessorApplicationSummary> applications)
         {
             foreach (var application in applications)
             {
                 var applicationVm = new ApplicationViewModel
                 {
                     ApplicationId = application.ApplicationId,
                     ApplicationReferenceNumber = application.ApplicationReferenceNumber,
-                    Assessor1 = application.Assessor1Name,
-                    Assessor2 = application.Assessor2Name,
+                    Assessor1Name = application.Assessor1Name,
+                    Assessor2Name = application.Assessor2Name,
                     ProviderRoute = application.ProviderRoute,
                     OrganisationName = application.OrganisationName,
                     Ukprn = application.Ukprn,
-                    SubmittedDate = application.SubmittedDate
+                    SubmittedDate = application.SubmittedDate,
+                    Assessor1UserId = application.Assessor1UserId,
+                    Assessor2UserId = application.Assessor2UserId
```

Comment:
you can maybe use Automapper here, they're like for like.  I'm fine leaving it as it is though.